tests/functests/instancetype_test.go

```diff
@@ -106,27 +114,31 @@ var _ = Describe("Common instance types func tests", func() {
 				testFn(virtClient, vm.Name)
 			}
 		},
-			Entry("[test_id:10738] Fedora", fedoraContainerDisk, "fedora",
+			Entry("[test_id:10738] Fedora", fedoraContainerDisk, map[string]string{"amd64": "fedora", "arm64": "fedora.arm64"},
 				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("fedora")}),
-			Entry("[test_id:10740] CentOS 7", centos7ContainerDisk, "centos.7",
+			Entry("[test_id:10740] CentOS 7", centos7ContainerDisk, map[string]string{"amd64": "centos.7"},
 				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("centos")}),
-			Entry("[test_id:10744] CentOS Stream 8", centosStream8ContainerDisk, "centos.stream8",
+			Entry("[test_id:10744] CentOS Stream 8", centosStream8ContainerDisk, map[string]string{"amd64": "centos.stream8"},
 				[]testFn{expectGuestAgentToBeConnected, expectSSHToRunCommandOnLinux("centos")}),
-			Entry("[test_id:10745] CentOS Stream 9", centosStream9ContainerDisk, "centos.stream9",
+			Entry("[test_id:10745] CentOS Stream 9", centosStream9ContainerDisk,
+				map[string]string{"amd64": "centos.stream9", "arm64": "centos.stream8"},
```

Comment:
```suggestion
				map[string]string{"amd64": "centos.stream9", "arm64": "centos.stream9"},
```

Wrong pref used for arm?